examples/echoserver/echoserver.c

```diff
@@ -110,6 +117,49 @@ static int dump_stats(thread_ctx_t* ctx)
     return wolfSSH_stream_send(ctx->ssh, (byte*)stats, statsSz);
 }
 
+#if defined(WOLFSSL_PTHREADS) && defined(WOLFSSL_TEST_GLOBAL_REQ)
+
+#define SSH_TIMEOUT 20
+
+static void callbackReqSuccess(WOLFSSH *ssh, void *buf, word32 sz, void *ctx)
```

Comment:
I didn't see where this callback is getting tested by the client responding. Is the client responding with its own Global Request?